design/apidsl/attribute.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/goadesign/goa/dslengine"
 )
 
+// Attribute can be used in: View, Type, Attribute (nested), Attributes (child),
```

Comment:
I guess conecptually, to me at the time of writing, nested indicated that it was able to be used within itself, but I see it is confusing to follow. I will just take out the parentheses I guess, unless you have any other ideas?